account_asset_transfer/wizard/account_asset_transfer.xml

```diff
@@ -62,6 +62,7 @@
                         name="transfer"
                         type="object"
                         class="oe_highlight"
+                        confirm="If journal entries are not found, asset transfer will use data from the asset/asset profile instead."
```

Comment:
I think you shouldn't use confirm here. Would be more user friendly to use the banner like in tier validation, and only when there is no account moves (may be some boolean field to help). The wording could be,

```
There are no journal entries on current asset(s), asset transfer will use account settings from the asset/asset profile instead.
```

Reply:
I mean the yellow banner should be at asset.transfer. Not at asset.